examples/pytorch/mnist/cnn_train.py

```diff
@@ -98,21 +97,20 @@ def train(args):
     """
     setup()
 
-    train_data = torchvision.datasets.ImageFolder(
-        root=args.training_data,
-        transform=transforms.ToTensor(),
+    train_dataset = datasets.MNIST(
```

Comment:
It is better to use a flag to download the MNIST dateset. Sometimes, it is very slow to download the MNIST dataset. For example:

```Python
if args.training_data:
    train_dataset = torchvision.datasets.ImageFolder(
          root=args.training_data,
          transform=transforms.ToTensor(),
else:
    train_dataset = datasets.MNIST(
        "./data", train=True, transform=transforms.ToTensor(), download=True
    )
```